json-schemas/protocol-rule-set.json

```diff
@@ -9,31 +9,57 @@
       "minItems": 1,
       "items": {
         "type": "object",
-        "required": [
-          "who",
-          "can"
-        ],
-        "additionalProperties": false,
-        "properties": {
-          "who": {
-            "type": "string",
-            "enum": [
-              "anyone",
-              "author",
-              "recipient"
-            ]
+        "anyOf": [
+          {
+            "required": [
+              "who",
+              "can"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "who": {
+                "type": "string",
+                "enum": [
+                  "anyone"
```

Comment:
Consider adding a test: 'should throw if `of` is specified for `anyone` rule'

Reply:
Added